api/policies/hasGameName.js

```diff
@@ -10,10 +10,12 @@ module.exports = function (req, res, next) {
   if (req.body.hasOwnProperty('gameName')) {
     if (typeof req.body.gameName === 'string') {
       // Could add restrictions on allowable game names
-      if (req.body.gameName.length > 0) {
-        return next();
+      if (!req.body.gameName.length) {
+        return res.badRequest({ message: 'Game name cannot be blank' });
+      } else if (req.body.gameName.length > 50) {
+        return res.badRequest({ message: 'Game name must contain fewer than 50 characters' })
```

Comment:
Technically misleading
```suggestion
        return res.badRequest({ message: 'Game name must cannot exceed 50 characters' })
```